src/content/docs/agile-handbook/heroing/what-is-a-hero.mdx

```diff
@@ -81,39 +86,101 @@ Common questions and requests include:
+
+We have three options for these tickets:
+
+1. Close it.
+2. Fix it.
+3. Backlog it.
+
+<Callout variant="important">
+Unlike the hero, we don't take any points out of our sprint for the sidekick work. If you're spending more than 60 minutes per day on customer feedback tickets, that's too much time! Please let your manager know.
+ </Callout>
+
+### Close customer feedback
+
+Many pieces of customer feedback simply aren't actionable. That's ok! Even if we can't act on a specific piece of feedback, we can use this feedback in aggregate to get an overall sense of how our docs site or a specific category of our docs site is landing with our readers.
+
+Here are some fictional examples of feedback we might not be able to take action on specifically:
+
+`These docs are bad.`
+
+or 
+
+`Help!`
+
+There's also feedback that falls into the category of nonsense or mistakes.
+
+Follow this process to close tickets that aren't actionable or don't make sense:
+
+1. Add a category label based on the URL, such as `doc_apm` or `doc_errors-inbox`.
+2. Set the Workflow status to **Closed** 
+3. Set the Resolution to **Cannot Reproduce**, **Filed by Mistake**, or **Won't Do** depending on what feels most appropriate.
+4. Add a brief note about why you closed it. Usually, "Not actionable" is plenty.
+
+### Fix customer feedback
+
+Sometimes, though rarely, a customer feedback ticket comes in that's highly actionable and is a quick fix, such as a broken link or an outdated image. 
+
+Follow this process to close these:
+
+1. Create a PR to address the issue.
+1. Add a category label based on the URL, such as `doc_apm` or `doc_errors-inbox`.
+1. Add a link to the PR.
+1. Work the PR to its merge at your leisure.
+1. Set the Jira ticket Workflow status to **Closed** 
+1. Set the Jira ticket Resolution to **Done**.
+
+### Backlog customer feedback
+
+Too many ticket backlogs have a larger quantity of tickets than can ever be meaningfully addressed. Our goal is to get our customer feedback backlog down to a small enough number that we can meaningfully and reasonably address them in a few sprints based on our current headcount and capacity. Because of this, we need to set a pretty high bar for adding a ticket to our backlog.
+
+Sometimes a piece of customer feedback is quite valid, but the scope of the work required to address it isn't something our team can't commit to right now. Sometimes a comment requires technological knowledge that we don't have. Sometimes a doc simply doesn't get enough traffic to warrant fixing a problem right away.
+
+If you're not sure whether a ticket belongs in our backlog, ask your manager or put it into the On Deck sprint to ensure it doesn't fall through the cracks.
+
+Use this process to put a ticket in the backlog:
+
+1. Write a user story.
+2. Write acceptance criteria.
+3. Write loose action items.
+4. Add any other detail about what you've learned.
+5. **Move** the ticket from the Customer Feedback to Story ticket type.
```

Comment:
Is it worth documenting the kind of opaque process for converting a feedback ticket into a story ticket?

Reply:
Probably not worth documenting? I'm willing to be wrong about this though!